### PR TITLE
Move system detail from FEC to this application

### PR DESCRIPTION
**Pull request by karelhala**
### Use federated modules to share system detail

Since we can now use federated modules in every environment let's share the system detail with it. This way we don't have to maintain the version of it in all environments, it's the same for all users on one environment. If a bug is found the fix is deployed just in this application and propagated to inventory via federetad modules.

### Usage
```JSX
import React from 'react';
import { useStore } from 'react-redux';
import AsyncComponent from '@redhat-cloud-services/frontend-components/AsyncComponent';

const AdvisorTab = () => {
    return <AsyncComponent
        appName="advisor"
        module="./SystemAdvisor"
        store={useStore()}
        customItnl
    />;
};
```

### CODE REVIEW

**Review by AllenBW — APPROVED** [review]
Okie dokie with this soon as travis is green, excited to rewrite the old version of react away 😜

**Review by AllenBW — CHANGES_REQUESTED** [review]
Nope i take that bad, wanta do the rewrite in this instead of somewhere else, will be throwing commits against this pr shortly

File: config/dev.webpack.config.js [hunk 1]
```diff
@@ -34,9 +34,9 @@ plugins.push(
       root: resolve(__dirname, '../'),
       exposes: {
         './RootApp': resolve(__dirname, '../src/AppEntry'),
-        './SystemDetail': resolve(
+        './SystemAdvisor': resolve(
```
Comment (karelhala) [inline config/dev.webpack.config.js:37]: I wouldn't rename the exported module name, we can keep the `./SystemDetail`. After all it's a system's detail, isn't it? You can keep the `src/SmartComponents/SystemAdvisor` file name though.

**Comment by karelhala** [comment]
@AllenBW looking good! Let's just have a consensus on the exported module. We can keep the name of the file, but I wouldn't change the module name. I am aiming to have all apps to expose `./SystemDetail` module. So importing it would be easy
```JS
<AsyncComponent
        appName="advisor" // if I'd want to pull from vulnerability I'd use `vulnerability` same for compliance and patch.
        module="./SystemDetail"
        store={useStore()}
        customItnl
    />;
```

File: config/dev.webpack.config.js (hunk 1 above)
Comment (AllenBW) [reply config/dev.webpack.config.js:37]: ah! gottcha on the logic why, onboard with that, will change it back

**Review by AllenBW — APPROVED** [review]
s not broken 🤯 s even 🧈 than 🐝 4️⃣

**Comment by AllenBW** [comment]
this work will also fix https://issues.redhat.com/browse/ADVISOR-1719 my thinkin was... why add old code when yah can do a lil work to add new code 🤷

**Pull request merged**